errors/wait-until-non-streaming.md

```diff
@@ -0,0 +1,19 @@
+# The waitUntil API has limited support for non-streaming functions
+
+## Why This Warning Occurred
+
+The [waitUntil](https://vercel.com/docs/functions/functions-api-reference#waituntil) method allows you to enqueue asynchronous tasks to be performed during the lifecycle of the request, and these tasks can continue to resolve even after the response has been written.
+
+This is possible because `waitUntil` takes advantage of [streaming functions](https://vercel.com/docs/functions/streaming#streaming-functions).
+
+When `waitUntil` is used in a non-streaming function, the behavior is limited and the response is delayed to be written after the enqueued asynchronous tasks have been resolved.
```

Comment:
> non-streaming function

I don't think users will always know streaming vs non-streaming. You can link them here: https://vercel.com/docs/functions/streaming#non-streaming-functions

Reply:
Also, could you just say something like the following to make it less passive:

```suggestion
When `waitUntil` is used in a [non-streaming function](https://vercel.com/docs/functions/streaming#non-streaming-functions), the behavior is limited and the function will delay the response until after the enqueued asynchronous tasks have been resolved.
```